### ISSUE TITLE
Method parameters documented inline are not present in documentation of overriding/implementing methods

### ISSUE THREAD

**Issue by LRad09** [doxygen/doxygen#6979]
Hi,

I'm using Doxygen version 1.8.15 and it appears that method parameters that are documented inline are not present in the documentation of methods overriding/implementing that method.

Here is a sample code:
```
/*!
   @brief Foo interface
*/
class IFoo
{
public:
   /*!
      @brief Function 1

      @param [in]    A     first parameter
      @param [in]    B     second parameter
      @param [in]    C     third parameter
   */
   virtual void Func1(int A
                     , float B
                     , std::string C
                     ) = 0;
   /*!
      @brief Function 2
   */
   virtual void Func2(int A /*!< [in] first parameter */
                     , float B /*!< [in] second parameter */
                     , std::string C /*!< [in] third parameter */
                     ) = 0;
};

/*!
   @brief Foo implementation
*/
class Foo : public IFoo
{
public:
   virtual void Func1(int A, float B, std::string C) override;
   virtual void Func2(int A, float B, std::string C) override;
};
```
And the resulting documentation:
Documentation for IFoo:
![doxygen_ifoo_doc](https://user-images.githubusercontent.com/50481138/57521631-1b42fa00-7321-11e9-9d8f-65933895ec25.png)

Documentation for Foo (implementation):
![doxygen_foo_imp_doc](https://user-images.githubusercontent.com/50481138/57521635-1c742700-7321-11e9-9293-887ee76cdfea.png)

Here is an archived containing everything required to reproduce this issue:
[inline_param_documentation.zip](https://github.com/doxygen/doxygen/files/3166095/inline_param_documentation.zip)

**Comment by albert-github** [comment]
The report is quite clear though when I use a default Doxyfile the Func2 in the class Foo is not shown at all so to be able to exactly reproduce your problem:
- Can you please attach a, small, self contained example (source+configuration file in a tar or zip) that allows us to reproduce the problem?  Please don't add external links as they might not be persistent.

A small tip when showing an image:
- show only the relevant part (in this case the left hand side)
or
- make the browser smaller before capturing the image

the result will be dat the image is a bit bigger and easier readable.

**Comment by LRad09** [comment]
Thank you for the quick reply and the tip!

I edited my original post with smaller and more relevant images and added a .zip file containing the source and the config file, which is the following:
[inline_param_documentation.zip](https://github.com/doxygen/doxygen/files/3166095/inline_param_documentation.zip)

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #6981

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of a release).

**Comment by LRad09** [comment]
Just download, compiled and tested your correction and it is working perfectly.
Thanks a lot!

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.16.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
